src/common/rivetsExtra.es6

```diff
@@ -60,10 +60,12 @@ rv.formatters['and-not'] = (vlaue, other) => vlaue && !other;
 rv.formatters['gt'] = (vlaue, other) => vlaue > other;
 /* rivets formatter for < operator  */
 rv.formatters['lt'] = (vlaue, other) => vlaue < other;
+/* rivets formatter to check if a variable has value */
+rv.formatters['has-value'] = (value) => value != null;
```

Comment:
`undefined`, `NaN` will still pass this check. 

Since you need a check for "truthy" numbers I think you could change this to `has-number` and find a way for it to check if the number is "truthy" and not 0

https://developer.mozilla.org/en-US/docs/Glossary/Truthy

Reply:
`undefined` won't pass this (since it's one `=`), but I don't know what to do with `NaN` values without being verbose. Maybe I can do something like this:

```javascript
rv.formatters['has-value'] = (value) => value === 0 ? true : Boolean(value)
```